apps/openmw/mwgui/windowmanagerimp.cpp

```diff
@@ -2033,6 +2034,11 @@ namespace MWGui
         return mTextColours;
     }
 
+    ToUTF8::FromType WindowManager::getEncoding()
+    {
+        return mEncoding;
+    }
```

Comment:
We could pass the encoding to the constructor of JournalWindow so as to not have to change WindowManager's external interface.